source/crud/tests/unified/bulkWrite-replaceOne-sort.yml

```diff
@@ -0,0 +1,94 @@
+description: BulkWrite replaceOne-sort
+
+schemaVersion: "1.0"
+
+createEntities:
```

Comment:
Unified tests typically use YAML anchors/aliases for entity IDs and names. This avoids duplication of the string literals later in the test, and ensures we get validation when converting YAML to JSON instead of possible runtime errors in test runners should an entity not be found.

Consider: https://github.com/mongodb/specifications/blob/c5dcae52a77d1ac90ca8c0cea7f371b73982062e/source/crud/tests/unified/aggregate-allowdiskuse.yml#L5-L21

The anchors defined in `createEntities` are referenced in subsequent entity definitions, as well as `initialData` and command expectations further down the test file.